### ISSUE TITLE
About train_sbi.py

### ISSUE THREAD

**Issue by Olive0407** [mapooon/SelfBlendedImages#23]
In train_sbi.py:
for step,data in enumerate(tqdm(train_loader)):
            img=data['img'].to(device, non_blocking=True).float()
            target=data['label'].to(device, non_blocking=True).long()
but in SBI_dataset : only return img_f,img_r. there is no label, could someone solve my problem.

**Comment by mapooon** [comment]
Labels are generated in `collate_fn()` of `SBI_Dataset`.
